core/src/test/java/org/kohsuke/stapler/test/AbstractStaplerTestV4.java

```diff
@@ -0,0 +1,9 @@
+package org.kohsuke.stapler.test;
+
+import org.kohsuke.stapler.AbstractStaplerTestBaseV4;
+
+/**
+ * Just to make the class visible in the test package.
```

Comment:
Not sure I understand. `AbstractStaplerTestBaseV4` is already `public`, so what is this intermediate type accomplishing?